cmd/blockchaincmd/convert.go

```diff
@@ -352,6 +354,7 @@ func InitializeValidatorManager(blockchainName, validatorManagerOwner string, su
 			extraAggregatorPeers,
 			aggregatorAllowPrivatePeers,
 			aggregatorLogger,
+			validatorManagerAddress,
```

Comment:
you need to add the validator manager address to the sidecar at some point. how is CLI going
to remember which is the manager for this blockchain and network? better for CLI to remember
it and not to ask the user to input the manager each time